src/20240707_AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity/IncreaseCCIPFacilitatorCapacity.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Increase CCIP Facilitator Capacity"
+author: "@karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-gho-arb-increase-ccip-facilitator-capacity/18169"
+snapshot: "Direct-to-AIP"
+---
+
+## Simple Summary
+
+This publication proposes increasing the CCIP Facilitator Capacity on Arbitrum from 1M units to 2.5M units via the Direct-to-AIP process.
+
+## Motivation
+
+GHO was successfully launched on Arbitrum and the CCIP GHO Facilitator Bucket Capacity is 65% utilized. With $3.41M of liquidity and a solid peg during launch, there is sufficient depth to facilitate increasing the CCIP GHO Facilitator Bucket Capacity from 1M to 2.5M in line with the initial risk parameters proposed prior to launch.
```

Comment:
@efecarranza Bucket Capacity currently is more than 65% currently no?

Reply:
Currently, it is at 100% atm. It was 65% at time we wrote the hackmd though. Good catch. 
Ref. Bottom of this dashboard. https://aave.tokenlogic.xyz/gho